client/my-sites/plan-features/index.jsx

```diff
@@ -288,6 +307,42 @@ class PlanFeatures extends Component {
 		} );
 	}
 
+	renderPopover() {
+		return (
+			<Popover
+				id="popover__plan-features"
+				isVisible={ this.state.showPopover }
+				context={ this.state.popoverReference }
+				position="right"
+				onClose={ this.closePopover }
+				className={ classNames(
+						'info-popover__tooltip',
+						'popover__plan-features'
+					) }
+				>
+					{ this.state.popoverDescripcion }
+			</Popover>
+		);
+	}
+
+	showPopover( el, popoverDescripcion ) {
+		const shouldHide = this.state.popoverReference === el &&
```

Comment:
> We could store the reference into <Item /> but we would be propagating the same reference to all of them and my idea here is simplify trying to avoid unnecessary code.

Yeah, if we want to reuse the same popover HTML structure and just change its positioning, we need a global reference.